nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -157,6 +157,20 @@ def __init__(self, ip_graph: InsertionPointGraph,
         for barred_node_key in self.ignored_node_keys + iteration_scope_node_keys:
             self._add_barrier_after_node(barred_node_key)
 
+    def get_node_keys_by_metatypes(self, metatype_list: List[Type[OperatorMetatype]]) -> List[str]:
```

Comment:
Refactor so that it accepts a single metatype, and returns the list of node keys that correspond to this single metatype. If you need to get nodes that correspond to either of a set of metatypes, just call this function multiple times and merge the corresponding lists.

Reply:
Done